src/pages/CardComplete.jsx

```diff
@@ -1,29 +1,51 @@
+import { useContext, useRef } from "react";
 import Card from "../components/Card";
-import { Link } from "react-router-dom";
+import { CardContext } from "../../providers/CardState/CardStateProvider";
+import { MONTH, YEAR } from "../constants/expirationDate";
+import Input from "../components/atomic-design-pattern/atom/Input";
+import Button from "../components/atomic-design-pattern/atom/Button";
+import { CardStorage } from "../util/cardStorage";
+
+export default function CardComplete({ onNext }) {
+  const { cardState } = useContext(CardContext);
+  const { cardNumber, expirationDate, cardOwnerName, alias } = cardState;
+  const aliasInputRef = useRef(null);
+
+  const onSubmitCardComplete = (event) => {
+    event.preventDefault();
+    const alias = aliasInputRef.current.value;
+    if (alias) {
+      CardStorage.changeCardInfo(cardNumber, "alias", alias);
+    }
+
+    onNext();
+  };
```

Comment:
c: [FormData API](https://developer.mozilla.org/en-US/docs/Web/API/FormData)를 사용하시면 ref로 요소를 참조하지 않아도 submit 이벤트에서 form data를 추출하실 수 있어요.

```suggestion
  const onSubmitCardComplete = (event) => {
    event.preventDefault();

    const formData = new FormData(e.target);
    
    // 추출한 formData를 이용한 로직...
  };
```